tests/Integration/Behaviour/Features/Context/Domain/Carrier/CarrierFeatureContext.php

```diff
@@ -388,6 +407,17 @@ public function carrierEditShouldThrowAnError(string $errorCode)
         );
     }
 
+    /**
+     * @Then carrier create should throw an error with error code :errorCode
+     */
+    public function carrierCreateShouldThrowAnError(string $errorCode)
```

Comment:
We don't need two methods to check the thrown exceptions There is already `carrierEditShouldThrowAnError` that should be more generic and be used in both creation and edition checks. You can add this additional step regexp `@Then carrier create should throw an error with error code :errorCode` on the existing `carrierCreateShouldThrowAnError` method (and rename it to something more generic if you want)